### DISCUSSION TITLE
Welcome to MusicPlayer Discussions!

### DISCUSSION THREAD

**Discussion by WednesdaySP** [WednesdaySP/MusicPlayer#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Varundeep111** [comment]
@WednesdaySP , In the readme file, how you have gathered all mockups in a single image, which tools you have used for this

**Comment by WednesdaySP** [comment]
@Varundeep111 I made this layout in figma using the plugin named "Artboard Mockups".
